src/Chapter2.hs

```diff
@@ -741,8 +773,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
 
+contains :: Int -> [[Int]] -> [[Int]]
+contains n xs = filter (\x -> elem n x) xs
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to this function and the lambda function as well  🙂 
But your implementation is already great 💯